src/utils/swapErrorToUserReadableMessage.tsx

```diff
@@ -12,11 +22,7 @@ export function swapErrorToUserReadableMessage(error: any): string {
     }
   }
```

Comment:
Can this initial check (for USER_REJECTED_REQUEST) be removed? Or are you keeping it for the deprecated non-UniversalRouter codepath?

Reply:
i think we should keep it so that `swapErrorToUserReadableMessage` is complete - this is the message it should return in this case.